src/Service/Ecr/.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
```

Comment:
I am missing the branch alias workflow. Could you add it please? See `src/Service/.template`

Reply:
doh, i copied `src/Service/.template` when was behind master.
might start again.